synapse/push/pusherpool.py

```diff
@@ -92,14 +92,14 @@ def remove_pushers_by_app_id_and_pushkey_not_user(self, app_id, pushkey,
                 yield self.remove_pusher(p['app_id'], p['pushkey'], p['user_name'])
 
     @defer.inlineCallbacks
-    def remove_pushers_by_user(self, user_id):
+    def remove_pushers_by_user_except_access_tokens(self, user_id, except_token_ids):
```

Comment:
Can we keep a version where we can delete all pushers too? Possibly making `except_token_ids` optional? As I'll need this when doing the logout stuff.